q2_taxa/assets/package.json

```diff
@@ -23,8 +23,8 @@
     "webpack-dev-server": "^1.16.1"
   },
   "scripts": {
-    "dev": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline",
-    "build": "./node_modules/webpack/bin/webpack.js"
+    "dev": "webpack-dev-server --inline",
```

Comment:
For whatever reason the CWD's `node_modules` is not added to my path when working with `npm`, so the relative path seemed to work instead. I don't have webpack-dev-server or webpack installed globally, just within the project. Also, this is a problem for me in qiime-studio, as well.